pic_database.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include('config/config.php');
+	include('include/db_query.php');
+
+	// InsertPic('Wes Hicks', '2018-05-25', 'TestPic', 'pictures/0f191f-wes-hicks-464614-unsplash.jpg', 'flavortext');
+	$pic = GetPic($_REQUEST['picID']);
+	//var_Dump($pic);
+	DisplayPic($pic);
+	// echo"<img src='pictures/0f191f-wes-hicks-464614-unsplash.jpg'alt='".$pic['flavor']."'>";
+
+
+	function InsertPic($photographer, $date, $title, $link, $flavor){
+			$result = dbQuery("
+				INSERT INTO pics (photographer, date, title, link, flavor)
+				VALUES('$photographer', '$date', '$title', '$link', '$flavor')
```

Comment:
It's not necessarily wrong to have this function take a date, but wouldn't the date always be right now? If so, I'd remove that as a parameter being passed to the function, and instead just take the current date and use that.